<launch>
  <node pkg="lidar_camera" name="lidar_camera" type="lidar_camera_node" output="screen" required="true">
    <remap from="cloud_in" to="/lidar0/velodyne_points"/>
    <remap from="image_in" to="/camera/rgb/image_raw"/>
    <remap from="camera_info" to="/camera/rgb/camera_info"/>
  </node>

  <node pkg="rviz" type="rviz" name="lidar_camera_viz" args="-d $(find lidar_camera)/launch/lidar_camera.rviz" required="false"/>
</launch>